src/tools/publish_toolstate.py

```diff
@@ -58,8 +58,10 @@ def read_current_status(current_commit, path):
     with open(path, 'rU') as f:
         for line in f:
             (commit, status) = line.split('\t', 1)
+
```

Comment:
I have no idea how these spaces got in here o.o

Should I remove them?

Reply:
Seems like a good idea to not make extra changes.